ui/packages/atlasmap/src/Atlasmap/Atlasmap.tsx

```diff
@@ -292,6 +290,7 @@ export const Atlasmap: FunctionComponent<IAtlasmapProps> = ({
   ]);
 
   const renderSidebar = useSidebar({
+    mappingExpressionEnabled: mappingExpressionEnabled,
```

Comment:
Sorry didn’t notice this before. You can get  this directly in the `useSidebar` hook, which uses `useAtlasmap`. No need to make it a prop.

Reply:
ah - good catch - see if this push is okay